frontend-instructor/src/components/navBar/NavItems.jsx

```
@@ -0,0 +1,26 @@
+  const handleLogOut = () => {
+    logOutInstructor();
+    toast.success("User Logged Out");
+  };
+
+  return (
+    <div className="flex items-center space-x-2">
+      {instructor.authenticated && (
+        <button
+          onClick={handleLogOut}
+          className="px-4 py-2 rounded-md text-sm font-medium transition-all bg-blue-600 text-white"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)